telegram/_webappdata.py

```diff
@@ -27,27 +27,26 @@ class WebAppData(TelegramObject):
     """Contains data sent from a `Web App <https://core.telegram.org/bots/webapps>`_ to the bot.
 
     Objects of this class are comparable in terms of equality. Two objects of this class are
-    considered equal, if their :attr:`data` and :attr:`button_text` are equal.
+    considered equal, if their :attr:`data` and :attr:`button_text` attributes are equal.
```

Comment:
I think this addition is superfluous, the user already knows that `data` and `button_text` are attributes

Reply:
I put this in because I felt, from a linguistic pov, it sounded more natural. Happy to revert, though.

Reply:
:thinking: Is that a :+1: for the linguistic improvement or for the *happy to revert*?

Reply:
Reverted.